### PR TITLE
Update contribute to include bdist_wheel

### PR DESCRIPTION
**Pull request by JackCaoG**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,6 +83,8 @@ We recommend you to use our prebuilt Docker image to start your development work
 
 * Build PyTorch
   ```Shell
+  # pytorch/xla requires pytorch wheel to be presented under pytorch/dist
+  python setup.py bdist_wheel
   python setup.py develop
```
Comment (vam-google) [inline CONTRIBUTING.md:88]: This line should be not needed. Or we need to explain why both are needed (the develop part is strictly speaking not needed to build or run pytorch/xla tests).

**Review by vam-google — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (JackCaoG) [reply CONTRIBUTING.md:88]: without the `develop` or `install` pytorch won't be install in the system. For the most part we need both pytorch and pytorch/xla installed to do the development.

**Comment by JackCaoG** [comment]
cc @alanwaketan

**Review by alanwaketan — APPROVED** [review]
LGTM.

**Pull request merged**